search/includes/classes/class-health-job.php

```diff
@@ -150,23 +150,35 @@ public function process_results( $results ) {
 					$result[ 'diff' ]
 				);
 
-				$this->send_alert( '#vip-go-es-alerts', $message, 2 );	
+				$this->send_alert( '#vip-go-es-alerts', $message, 2, $result['type'] );
```

Comment:
I just realized there's an edge case here - if the if two different "entities" (like Post, User, or Term) have the same "type" registered here, they'll conflict. In practice this likely won't happen (so maybe not worth fixing). If there was a CPT called `user` or something it might happen.

I guess for completeness, we could pass through the entity + type to unique identify it.
```suggestion
				$this->send_alert( '#vip-go-es-alerts', $message, 2, "{$result['entity']}:{$result['type']}" );
```